pkg/reconciler/taskrun/taskrun_test.go

```diff
@@ -122,601 +68,60 @@ var _ = Describe("TaskRun Reconciler Tests", func() {
 				},
 			}
 
-			Expect(extractPlatform(tr)).To(Equal("linux/amd64"))
-		})
-
-		It("should extract platform from TaskRun with multiple parameters", func() {
-			tr := &pipelinev1.TaskRun{
-				Spec: pipelinev1.TaskRunSpec{
-					Params: []pipelinev1.Param{
-						{Name: "OTHER_PARAM", Value: *pipelinev1.NewStructuredValues("other_value")},
-						{Name: PlatformParam, Value: *pipelinev1.NewStructuredValues("linux/arm64")},
-						{Name: "ANOTHER_PARAM", Value: *pipelinev1.NewStructuredValues("another_value")},
-					},
-				},
-			}
-
-			Expect(extractPlatform(tr)).To(Equal("linux/arm64"))
-		})
-
-		It("should return first occurrence when multiple PlatformParam parameters exist", func() {
-			tr := &pipelinev1.TaskRun{
-				Spec: pipelinev1.TaskRunSpec{
-					Params: []pipelinev1.Param{
-						{Name: "OTHER_PARAM", Value: *pipelinev1.NewStructuredValues("other_value")},
-						{Name: PlatformParam, Value: *pipelinev1.NewStructuredValues("linux/amd64")},
-						{Name: "MIDDLE_PARAM", Value: *pipelinev1.NewStructuredValues("middle_value")},
-						{Name: PlatformParam, Value: *pipelinev1.NewStructuredValues("linux/arm64")},
-						{Name: PlatformParam, Value: *pipelinev1.NewStructuredValues("linux/s390x")},
-					},
-				},
-			}
-
-			Expect(extractPlatform(tr)).To(Equal("linux/amd64")) // Should return the first occurrence
-		})
-
-		It("should return error when TaskRun has nil parameters", func() {
-			tr := &pipelinev1.TaskRun{
-				Spec: pipelinev1.TaskRunSpec{
-					Params: nil,
-				},
-			}
-
-			Expect(extractPlatform(tr)).Error().To(MatchError(errFailedToDeterminePlatform))
-		})
-
-		It("should return error when TaskRun has no parameters", func() {
-			tr := &pipelinev1.TaskRun{
-				Spec: pipelinev1.TaskRunSpec{
-					Params: []pipelinev1.Param{},
-				},
-			}
-
-			Expect(extractPlatform(tr)).Error().To(MatchError(errFailedToDeterminePlatform))
+			platform, err := extractPlatform(tr)
+			Expect(err).NotTo(HaveOccurred())
+			Expect(platform).To(Equal("linux/amd64"))
```

Comment:
Why are we reverting this?
```suggestion
			Expect(extractPlatform(tr)).To(Equal("linux/amd64"))
```